src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -218,6 +219,20 @@ std::pair<ov::Tensor, ov::Tensor> InputsEmbedder::IInputsEmbedder::get_inputs_em
     OPENVINO_THROW("This model does not support token_type_ids.");
 }
 
+std::pair<ov::Tensor, ov::Tensor> InputsEmbedder::IInputsEmbedder::get_inputs_embeds_with_token_type_ids(
+    const std::string& prompt,
+    const std::vector<EncodedImage>& images,
+    const std::vector<ov::genai::EncodedVideo>& videos,
```

Comment:
```suggestion
    const std::vector<EncodedVideo>& videos,
```
already in `ov::genai` namespace